### PR TITLE
Fix distcheck configure args

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

**Comment by peternewman** [comment]
> I am a bit confused about the difference between `./configure <args>` and `DISTCHECK_CONFIGURE_FLAGS`, but it is clearly some internal working of Automake.

I think the reason is because during distcheck it essentially does:
./configure
make dist

Then on the dist file does:
./configure
make
make check

Or something like that. Given the second configure happens as an internal process, you can't tell it what args to use traditionally, and I guess you might want different args for the two configure runs, hence the env var.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -92,11 +92,19 @@ jobs:
       - name: Set configure arguments
         run: |
           echo "GH_OLA_CONFIGURE_ARGS=${{ matrix.configure-args }}" >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (peternewman) [reply .github/workflows/build.yaml:94]: Yeah that sounds like it should work. We should probably also backport the comment to the Travis file too.

Do you fancy picking this one up?

[the remark this answers is not in the document]
Comment (DaAwesomeP) [reply .github/workflows/build.yaml:94]: Sure, if that will be faster I can checkout your branch and open a new pull, but it is simple enough to change it here too. Up to you.

[the remark this answers is not in the document]
Comment (DaAwesomeP) [reply .github/workflows/build.yaml:94]: See #1885

**Pull request merged**